fetch/main.js

```diff
@@ -0,0 +1,28 @@
+const logJSONData = async () => {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const dataJson = await response.json();
```

Comment:
You should check `response.ok` before getting the `json` since the `json` may be invalid or missing if the response is a 500.